build.gradle

```diff
@@ -1208,6 +1208,18 @@ task checkBasicStyle(group: 'Format') {
 
 assemble.mustRunAfter(clean)
 
+/**
+ * Creates a new task named `fastAssemble` that executes
+ * the `assemble` task without generating any Javadoc.
+ */
+task fastAssemble(group: 'Build') {
+    tasks.withType(Javadoc) {
```

Comment:
```suggestion
task fastAssemble(group: 'Build') {
    description 'Assembles the outputs of this project without generating any Javadoc'
    tasks.withType(Javadoc) {
```

Reply:
This should make sure that something shows up in `./gradlew tasks`. Maybe mention both javadoc and source jars?

Reply:
Thanks for pointing that out! Added `sources.jar` to documentation.